src/rawchannel.cc

```diff
@@ -375,13 +376,16 @@ class RawChannel : public Nan::ObjectWrap
     CHECK_CONDITION(info[0]->IsObject(), "First argument must be an Object");
     CHECK_CONDITION(hw->IsValid(), "Invalid channel!");
     struct canfd_frame frameFD;                         
-    frameFD.flags = 0; // Flags not used in this function
+    //frameFD.flags = 0; // Flags not used in this function
 
     v8::Local<v8::Context> context = info.GetIsolate()->GetCurrentContext();
     v8::Local<v8::Object> obj = Nan::To<Object>(info[0]).ToLocalChecked();
 
     frameFD.can_id = obj->Get(context, id_symbol).ToLocalChecked()->ToUint32(context).ToLocalChecked()->Value();
 
+    //set CAN frame flags, mask only bit 0..2 (CANFD_BRS, CANFD_EFI and CANFD_FDF) - see .../include/linux/can.h
+    frameFD.flags = 0b111 & obj->Get(context, fdflags_symbol).ToLocalChecked()->ToUint32(context).ToLocalChecked()->Value();
```

Comment:
As the flag might not be used by everybody (contrary to e.g. CAN ID) please check if the parameter was even passed by the caller.